src/eddymotion/estimator.py

```diff
@@ -132,7 +136,12 @@ def fit(
                         pbar.set_description_str(
                             f"Pass {i_iter + 1}/{n_iter} | Fit and predict b-index <{i}>"
                         )
-                        data_train, data_test = dwdata.logo_split(i, with_b0=True)
+                        dwframe = np.asanyarray(dwdata.dataobj[..., i])
+                        bframe = np.asanyarray(dwdata.gradients[..., i])
+                        data_train, data_test = logo_split(
+                            dwdata, dwframe, bframe, i, with_b0=True
```

Comment:
Function signature should not change (except for making the data object the first argument and removing `with_b0` which someone (me) added on a gloomy day.